csrc/python_frontend/python_bindings.cpp

```diff
@@ -171,6 +184,89 @@ std::vector<std::optional<bool>> computeContiguity(
   return contiguity;
 }
 
+// [ Note stride order and contiguity vector ]
+//
+// `stride order` vector corresponds to the order for each logical domain in
+//     physical memory;
+// `contiguity` vector to whether or not indexing could be collaped
+//     corresponding to each physical domain;
+//
+// e.g. Given size and stride as follow:
+//   sizes   = [2, 1, 3, 1, 4, 3]
+//   strides = [12, 4, 4, 4, 1, 0]
+// we would compute stride order as: [5, 4, 3, 2, 1, 0]. Since the original
+// stride is in descending order. Note that there's more than one way to define
+// a stride order when we have equal strides. In the context of index
+// collapsing, how we resolve that shouldn't matter, hence we just go with
+// preserving their original order. Similarly, we compute contiguity as: [True,
+// None, True, None, True, None], Since the physical order is the same as the
+// logical order, this one is trivial to compute.
+//
+// e.g. Given size and stride as follow:
+//   sizes   = [2, 3, 1, 5, 4]
+//   strides = [28, 4, 14, 0, 1]
+// stride_order would be: [4, 2, 3, 0, 1], marking the order of strides in the
```

Comment:
It's hard for me to argue for what I put in there since I also found it confusing for myself. But I slowly get used to it.

I don't have strong opinions on how we should handle this. But I would like to keep it consistent with potential integration. i.e. whatever framework decided to represent it. :wink: